src/lib/tvgTvgHelper.h

```diff
@@ -0,0 +1,82 @@
+#ifndef _TVG_STANDARD_HELPER_H_
```

Comment:
Is there any reason this Helper header file should be in src/lib?
I think it should be in src/loaders/tvg if it's only used in tvgTvgLoader.
And if you move this header file, I think proper name of this file is tvgTvgLoaderHelper.h.

Reply:
it is also used by the saver, but I need a moment to decide where the saver will be implemented

Reply:
Renamed `_TVG_STANDARD_HELPER_H_` into `_TVG_TVG_HELPER_H_` and added copyright header